packages/modal/src/types.js

```diff
@@ -1,6 +0,0 @@
-export const types = Object.freeze({
-  STANDARD: "standard",
-  ALTERNATE: "alternate"
-});
-
-export const availableTypes = Object.freeze(Object.values(types));
```

Comment:
Rename this to `AVAILABLE_TYPES`, but to avoid a breaking change also export `availableTypes` as an alias with a comment to remove.

Reply:
Addressed by 1c5bfed9f3f66859165a53246bd89c0b253122f0